kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/UserTaskServiceBase.java

```diff
@@ -85,11 +86,15 @@ public void activate(String containerId, Number taskId, String userId) {
     }
 
     public void claim(String containerId, Number taskId, String userId) {
-
         userId = getUser(userId);
         logger.debug("About to claim task with id '{}' as user '{}'", taskId, userId);
-        userTaskService.claim(taskId.longValue(), userId);
-
+        userTaskService.claim(containerId, taskId.longValue(), userId);
+    }
+    
+    public void claim(String containerId, Collection<Long> taskIds, String userId) {
```

Comment:
@fjtirado failures related this happens you need in the service base to set as Number not as Long

Reply:
I think the failure is caused by the typo detected by Gonzalo https://github.com/kiegroup/droolsjbpm-integration/pull/2518#discussion_r655945914
Lets wait this round to see,